src/System.IO/src/System/IO/MemoryStream.cs

```diff
@@ -474,41 +474,48 @@ public override int ReadByte()
             return _buffer[_position++];
         }
 
-
-        public override Task CopyToAsync(Stream destination, int bufferSize, CancellationToken cancellationToken)
+#if netcoreapp11
```

Comment:
Your correct this is working because it is directly compiling against System.Private.CoreLib. However I wouldn't say that makes it default to netcoreapp1.1 because it is still being packaged as netstandard1.7. For now I think we can live with this because we are soon going to overhaul our packages and this will become a non-issue. 

FYI @ericstj